### ISSUE TITLE
Move tourism=guest_house and tourism=chalet to z18+

### ISSUE THREAD

**Issue by lakedistrictOSM** [gravitystorm/openstreetmap-carto#3448]
These tend to be small features (normally house sized or smaller) and currently render too early, sometimes hiding more useful information on the map. 

There are more crowded areas than this, but this is just one area where rendering would be improved:
![image](https://user-images.githubusercontent.com/20193716/46822842-78bf9a00-cd84-11e8-9e11-5fc0f1b296bc.png) https://www.openstreetmap.org/#map=17/54.42957/-2.96538

Blackpool: https://www.openstreetmap.org/#map=17/53.80262/-3.05464
![image](https://user-images.githubusercontent.com/20193716/46823766-f8e6ff00-cd86-11e8-96e0-8b79510c781a.png)

![image](https://user-images.githubusercontent.com/20193716/46823453-1ff10100-cd86-11e8-97c2-ff55548379e2.png)
https://www.openstreetmap.org/#map=17/46.08727/6.66280

z17 is probably too early for tourism=hostel in some places as well

**Comment by Tomasz-W** [comment]
Yeah, it's hurting my eyes also on the polish coast of the Baltic Sea:
https://www.openstreetmap.org/#map=17/54.09418/15.07732
https://www.openstreetmap.org/#map=17/54.21161/15.74899
https://www.openstreetmap.org/#map=17/54.60542/18.80378

So fixing it both with https://github.com/gravitystorm/openstreetmap-carto/issues/3410 would be reasonable.

**Comment by polarbearing** [comment]
Again it is a matter of density. If it is the only guesthouse in the area, it is harder to find when it is moved to higher zoom. 
Seeing so many icons characterises the area. When motortravelling in the US, I found the motels typically congregated in a particular area of the town, and so are the guesthouses along the coast or the restaurants in a particular district of the city in the examples above.

**Comment by Adamant36** [comment]
I was already thinking things like this needed a blue dot at z17. So I support 
@Tomasz-W's suggestion.

**Comment by Tomasz-W** [comment]
What do you think about another accomodation places started being rendered from z17?:
* tourism=chalet
* tourism=hostel
* tourism=hotel
* tourism=motel

At the one hand it may be not fair for both accomodation providers and tourists to prioritize one accomodation type over another ones, but at the other hand some of them gives higher standard and also (as I know) there are no examples of them cluttering the map. If there are some, @polarbearing please post links with example locations due to your comment:

> When motortravelling in the US, I found the motels typically congregated in a particular area of the town

@Adamant36 We can't use dot here, because _blue_ is used not only for accomodation, but also for transport so it would be confusing (see https://github.com/gravitystorm/openstreetmap-carto/issues/3395)

**Comment by Adamant36** [comment]
@tomasz-W, Oh. I thoughts that what you were saying. My bad. Transports a square though isn't it? I know bus stops are at least. Not to mention hotels etc icons are usually over buildings/not in the middle of the road. Plus, id like to see kind of standardization. It doesn't make sense to have dots/squares for everything else besides accommodations at z17, some things rendered at z18, then others rendered at z19. Especially if you go by @kocio-pl's size thing as some hotels are rather big. I think accommodations do clutter it up in some places at z17, but there should still be an indication at that level that they are there.

**Comment by lakedistrictOSM** [comment]
z17 if fine for hotels, since they tend to be bigger, but z17 is too early too start rendering icons for smaller accommodation providers as shown in the examples above.

If they are unusually large or remote, then the name or addr:housename will probably show up at z17, giving an idea of what's there.

**Comment by kocio-pl** [comment]
I don't feel this problem is that big, but I would start with review of all current accommodation objects, not just these 2 alone:

https://github.com/gravitystorm/openstreetmap-carto/blob/4b9771de1c7242f2026a5fd40d19b8599c9c19c5/amenity-points.mss#L2322-L2332

As you can see alpine huts and wilderness huts are exaggerated a lot, since both are outdoor facility, and I think this is OK.

Next thing would be to check how would look the rendering of blue dots near bus stop blue squares, to make sure if there would be a visual problem or maybe not.

**Comment by HolgerJeromin** [comment]
btw this list in the `mss` should be sorted somehow. Name or zoom level but not unsorted :)

**Comment by HolgerJeromin** [comment]
sometimes alphabetically is not the best, but a category:
#1694

**Comment by matkoniecz** [comment]
I think that every example provided is OK. I think that it is fine to display many guest houses in places that contain primarily guest houses (and AFAIK Blackpool and other examples are places like this).

**Comment by matkoniecz** [comment]
> As you can see alpine huts and wilderness huts are exaggerated a lot, since both are outdoor facility, and I think this is OK.

Yes, this is one of rare cases where "rural areas are less dense and objects can be displayed earlier" was easy to implement.

**Comment by Prince-Kassad** [comment]
Right now we use blue dots for offices, using blue dots also for accomodation would create massive confusion.

With the current definition given according to the wiki, tourism=guest_house is basically tourism=hotel but without permanently staffed reception desk. I suspect from the pictures above that these cities do not really have 100 different guest houses right next to each other, which would be something along the line of 1000 rooms, which is on scale with summer destinations like Palma.

The real problem is, there is no area rendering for tourism=chalet, which is why people keep tagging every single house of a large resort, e. g. Center Parcs. It should render similar to camp_site.

**Comment by Adamant36** [comment]
> I suspect from the pictures above that these cities do not really have 100 different guest houses right next to each other, which would be something along the line of 1000 rooms, which is on scale with summer destinations like Palma.

Here's a link to the area on the map. It looks like it's right next to a large beach and that they are in fact 100 different guest houses. It doesn't seem to be a large resort either. Although, that's just from going off the satellite photos. So more research is obviously needed. 

https://www.openstreetmap.org/#map=19/53.80295/-3.05384

**Comment by Adamant36** [comment]
> The real problem is, there is no area rendering for tourism=chalet, which is why people keep tagging every single house of a large resort

If this is still an issue and @Prince-Kassad et. el think area rendering is a solution, I'd be willing to do some test renderings if someone has a suggestion of how to render it and wants me to.

**Comment by jeisenbe** [comment]
The area of a tourism=chalet should be quite small and can be mapped as a building which is rendered, but I think the reference may have been the lack of an area rendering for large hotel and resort accomodations, which can cover extensive areas. The last example probably shows this: https://www.openstreetmap.org/#map=17/46.08727/6.66280 - with tourism=chalet on each building instead of mapping the whole area as one feature.

(We don't render `leisure=resort` at all currently, see #2290 - though we do render `leisure=beach_resort` oddly enough)

However, shouldn't tourism accomodations be mapped within a landuse=commercial area? I would think that would be the most appropraite area rendering. While we could render tourism accomodation areas with the commercial fill color, this might make mappers forget to add the `landuse=` as well.

**Comment by Prince-Kassad** [comment]
> However, shouldn't tourism accomodations be mapped within a landuse=commercial area?

Our wiki pages are unfortunately not very clear what a landuse=commercial actually is, but I'm assuming what is meant is a CBD, in which case that wouldn't fit very well for a hotel resort. We already exclude shops and retail centers from the definition of commercial by giving them their own landuse. I do think recreation-green is a more fitting color than commercial-red, personally, especially when you think of places like the Caribbean with miles and miles of hotel resorts.

**Comment by jeisenbe** [comment]
"Commercial landuse mainly deals with services and trade (tertiary sector). Such area may consist of offices, administration, laboratories, warehouses (logistics park), hotels, car repair stations and their associated infrastructure" - https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dcommercial

I believe this is fairly clear. Commercial is not only central business districts (which often would be landuse=retail), but any area that is developed with buildings such as offices and hotels, rather than retail, industrial or residential buildings.

**Comment by dch0ph** [comment]
A conservative option would be delaying rendering the name of "small providers" (`guest_house`, `chalet`) to Z18. Currently both symbol and name appear for all at Z17, which is a bit overwhelming.

This would still show the existence of accommodation at Z17 (helpful in rural areas), but reduce visual clutter (and collisions).

It is helpful to give more emphasis to "full service providers", such as hotels, by showing the name at Z17.
